packages/charts/src/charts/bar/utils/chart.utils.ts

```diff
@@ -328,6 +328,7 @@ export const generateHorizontalStackedBars = ({
         width,
         height: barHeight,
         color: getPaletteColor(keys.indexOf(keyName), colors),
+        colorOutOfRange: !colors[index],
```

Comment:
Could You please add additional test case that generates a bar with `colorOutOfRange` set to `true`.

Reply:
Tests added